scripts/patterns-reorg/update_internal_links.py

```diff
@@ -52,10 +52,11 @@ def update_link(markdown: str, folder: str, link: str, prefix: str) -> str:
         else:
             search_key = f"{folder}/{link_split[-1]}"
 
+    search_key = f"/{search_key.removeprefix('/')}"
```

Comment:
I used the `removeprefix('/')` because we have links like `/verify` or `verify/index`. Removing the slash to all links allows us to add it back to all the links safely without having to use a conditional for the former case which would end up being `//verify` otherwise.